scripts/setup_system.sh

```diff
@@ -35,6 +35,7 @@ sudo apt-get -y install \
     net-tools \
     git-lfs \
     bc \
```

Comment:
i think you should add gettext-base here as well for the users that would use a local registry on bare metal.

Reply:
added.